src/backend/distributed/deparser/ruleutils_14.c

```diff
@@ -6437,6 +6443,139 @@ get_func_expr(FuncExpr *expr, deparse_context *context,
 			appendStringInfoString(buf, "VARIADIC ");
 		get_rule_expr((Node *) lfirst(l), context, true);
 	}
+
+	appendStringInfoChar(buf, ')');
+}
+
+/*
+ * get_proc_expr			- Parse back a CallStmt node
+ */
+static void
+get_proc_expr(CallStmt *stmt, deparse_context *context,
+			  bool showimplicit)
+{
+	StringInfo	buf = context->buf;
+	int		argIndex = 0;
+	bool		use_variadic;
```

Comment:
is this new code or (partially) copied from PG? The indentation looks a bit weird, but we usually leave PG's code style in place when copying code to make the diff easier. This file therefore does not apply our usual style & indendation rules when running citus_indent, but we'd typically still follow them when adding new code.

A few minor comments below (mainly fyi).

Reply:
After looking at get_func_expr a bit more, it's probably midway in between PG and Citus code, so I'm fine keeping current style.

Reply:
Thanks @marcocitus  for the review, when I looked at the PR I was confused why this is in zigzag mode. This is how I see it in my VIM 
![image](https://user-images.githubusercontent.com/44680808/136246508-241a02aa-b9f4-45be-8bfe-bef95cdbf2bd.png)
Using ONLY tabs to align them. I will fix the other minor comments

Reply:
Which one Citus use?
Oid<blank>Var;
or
Oid<tab>Var;

Reply:
Regarding this >>is this new code or (partially) copied from PG?

Most of the code, I reused from the same file for both the syntax and snippets, as you can see, single IF has no braces
![image](https://user-images.githubusercontent.com/44680808/136271500-21b4ed4a-5316-4263-8ae2-3887637af29e.png)
and switch-case has no braces either
![image](https://user-images.githubusercontent.com/44680808/136271647-aeea9491-d8ef-4af0-a24c-d9e93e93ce18.png)
But I just checked other files, such as distribute_object_ops.c, followed the syntax